src/store/lokapi.ts

```diff
@@ -134,6 +138,13 @@ export var moduleLokAPI = {
       });
       state.recipientHistory = [...new Set(filtered)];
       state.thisWeektransactions = trs
+    },
+    async setBackends(state:any) {
+      try {
+        state.backends = await lokApiService.getBackends()
+      } catch (err:any) {
+        console.error('Error getting currency backends')
+      }
     }
```

Comment:
In Vuejs topology, the mutations are only responsible of mutating the store state. They should not contain any call to external services. That's what actions are here for:
[https://vuex.vuejs.org/guide/mutations.html](https://vuex.vuejs.org/guide/mutations.html)
[https://vuex.vuejs.org/guide/actions.html](https://vuex.vuejs.org/guide/actions.html)